File: Javascript/A_NivelIniciante/Exercicio_1040.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var lines2 = lines[0].split(" ");
var media = (lines2[0]*2 + lines2[1]*3 + lines2[2]*4 + lines2[3]*1)/10
console.log("Media: " + media.toFixed(1));
if(media>=7.0){
    console.log("Aluno aprovado.")
}else if(media<5.0){
    console.log("Aluno reprovado.")
}else if(media>=5.0 && media<=6.9){
    let exame= lines[1]*1
    console.log("Aluno em exame.");
    console.log("Nota do exame: " + exame.toFixed(1))
    var media_final = (exame + media)/2
    if(media_final>=5.0){
    console.log("Aluno aprovado.");
    }else{
    console.log("Aluno reprovado.");    
    }
    console.log("Media final: " + media_final.toFixed(1))
}

File: Java/A_NivelIniciante/Exercicio_1040.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package A_NivelIniciante;

import java.util.Scanner;

/**
 *
 * @author Scarlxrd2112
 */
public class Exercicio_1040 {

    public static void main(String[] args) {
        float N1, N2, N3, N4, media, exame, media_final;
        Scanner input = new Scanner(System.in);
        String [] ns = input.nextLine().split(" ");
        N1 = Float.parseFloat(ns[0]);
        N2 = Float.parseFloat(ns[1]);
        N3 = Float.parseFloat(ns[2]);
        N4 = Float.parseFloat(ns[3]);

        media = ((N1 * 2) + (N2 * 3) + (N3 * 4) + (N4 * 1)) / 10;
        System.out.printf("Media: %.1f\n", media);
        if (media >= 7.0){
            System.out.print("Aluno aprovado.\n");

        } else if (media >= 5.0 && media <= 6.9) {
            System.out.print("Aluno em exame.\n");
            exame = Float.parseFloat(input.nextLine());
            System.out.printf("Nota do exame: %.1f\n", exame);
            media_final = (media + exame) / 2;

            if (media_final >= 5.0) {
                System.out.print("Aluno aprovado.\n");

            } else {
                System.out.print("Aluno reprovado.\n");

            }

            System.out.printf("Media final: %.1f\n", media_final);

        } else if (media < 5.0) {
            System.out.print("Aluno reprovado.\n");

        }

    }
}
